src/main/java/eod/card/concrete/normal/FightBack.java

```diff
@@ -82,9 +80,7 @@ public void onEventOccurred(GameObject sender, Event event) {
             if(event instanceof TargetedEvent) {
                 TargetedEvent e = (TargetedEvent) event;
                 if(e.getTarget() == watching) {
-                    //TODO: find a way to use player or game (i.e. EffectHandler) to do this
-                    Damageable attacker = (Damageable) e.getAttacker();
-                    attacker.damage(4);
+                    new Damage(new DamageParam(4), Effect.HandlerType.Rival).on((Damageable) e.getAttacker());
```

Comment:
Use static Damage() method please.
The problem here is that we cannot find EffectExecutor, so the effect won't be executed.
I suggest that we add the TODO back, and use the old way to make it work.

Reply:
fixed. I think the new version will work.